src/hooks.ts

```diff
@@ -77,10 +80,11 @@ function areCallResultsEqual(a: CallResult[], b: CallResult[]) {
 function useMultichainCallsDataSubscription(
   context: MulticallContext,
   chainToCalls: Record<number, Array<Call | undefined>>,
-  blocksPerFetch = 1
+  blocksPerFetch?: number
```

Comment:
Why not change this to `ListenerOptions`?

Reply:
Sgtm done